test/recipes/03-test_fipsinstall.t

```diff
@@ -359,6 +359,36 @@ SKIP: {
        "fipsinstall fails when the asymmetric cipher result is corrupted");
 }
 
+# corrupt ML-KEM tests
+SKIP: {
+    skip "Skipping ML_KEM corruption tests because of no ML-KEM in this build", 3
+        if disabled("ml-kem") || disabled("fips-post");
+
+    ok(!run(app(['openssl', 'fipsinstall', '-out', 'fips.cnf', '-module', $infile,
+                '-provider_name', 'fips', '-mac_name', 'HMAC',
+                '-macopt', 'digest:SHA256', '-macopt', "hexkey:$fipskey",
+                '-section_name', 'fips_sect',
+                '-corrupt_desc', 'KEM_Encap',
+                '-corrupt_type', 'KAT_KEM'])),
+       "fipsinstall fails when the ML_KEM encapsulate result is corrupted");
+
+    ok(!run(app(['openssl', 'fipsinstall', '-out', 'fips.cnf', '-module', $infile,
+                '-provider_name', 'fips', '-mac_name', 'HMAC',
+                '-macopt', 'digest:SHA256', '-macopt', "hexkey:$fipskey",
+                '-section_name', 'fips_sect',
+                '-corrupt_desc', 'KEM_Decap',
+                '-corrupt_type', 'KAT_KEM'])),
+       "fipsinstall fails when the ML_KEM decapsulate result is corrupted");
+
+    ok(!run(app(['openssl', 'fipsinstall', '-out', 'fips.cnf', '-module', $infile,
```

Comment:
Does this one work? Isnt this a PCT?

Reply:
It does both a PCT and a CAST.  This is failing the CAST.